api/MediaSource.json

```diff
@@ -663,50 +670,113 @@
           }
         }
       },
-      "removeSourceBuffer": {
+      "sourceclose_event": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/removeSourceBuffer",
-          "spec_url": "https://w3c.github.io/media-source/#dom-mediasource-removesourcebuffer",
+          "description": "<code>sourceclose</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/sourceclose_event",
+          "spec_url": "https://w3c.github.io/media-source/#dfn-sourceclose",
           "support": {
-            "chrome": {
-              "version_added": "23"
-            },
-            "chrome_android": {
-              "version_added": "25"
-            },
+            "chrome": [
+              {
+                "version_added": "53"
+              },
+              {
+                "version_added": "31",
+                "version_removed": "53",
+                "partial_implementation": true,
+                "notes": "The <code>onsourceclose</code> event handler property is not supported."
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "53"
+              },
+              {
+                "version_added": "31",
+                "version_removed": "53",
+                "partial_implementation": true,
+                "notes": "The <code>onsourceclose</code> event handler property is not supported."
+              }
+            ],
             "edge": {
-              "version_added": "12"
-            },
-            "firefox": {
-              "version_added": "42"
-            },
-            "firefox_android": {
-              "version_added": "41"
+              "version_added": "17"
             },
+            "firefox": [
+              {
+                "version_added": "87"
+              },
+              {
+                "version_added": "42",
+                "partial_implementation": true,
+                "notes": "The <code>onsourceclose</code> event handler property is not supported."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "87"
+              },
+              {
+                "version_added": "42",
```

Comment:
I think Firefox only has this problem with this specific event. Added version_removed though.

Reply:
Ah yes, this was a case of just `onsourceclose` being missing, since there was a typo calling it `onsourceclosed`, I filed that bug :) https://bugzilla.mozilla.org/show_bug.cgi?id=1689222

Reply:
Oh, but also the version should be 41 here.